pkg/storage/enginepb/mvcc3.proto

```diff
@@ -170,6 +170,10 @@ message MVCCValueHeader {
   // to stale reads.
   util.hlc.Timestamp local_timestamp = 1 [(gogoproto.nullable) = false,
     (gogoproto.casttype) = "github.com/cockroachdb/cockroach/pkg/util/hlc.ClockTimestamp"];
+
+  // ImportEpoch identifies the number of times a user has called IMPORT
+  // INTO on the table this key belongs to when the table was not empty.
```

Comment:
nit: consider briefly explaining its purpose, i.e. that it's used for cancelling an import following a restore.